gcloud/orb.yaml

```diff
@@ -397,6 +397,11 @@ jobs:
         description: >
           Extra flags to pass to docker build.
         type: string
+      custom-checkout:
```

Comment:
Hmm, maybe call this `repo` (or `git_repo`)? This name implies (to me) that we should provide a full checkout command rather than just the name of the repo to-be-checked-out.